config/boot.rb

```diff
@@ -3,4 +3,4 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
-require 'bootsnap/setup' # Speed up boot time by caching expensive operations.
+# require 'bootsnap/setup' # Speed up boot time by caching expensive operations.
```

Comment:
Here i just turned off Bootsnap for now. Let's get it running in GitHub Actions and then we can re-enable it in follow-up once we figure it out. The issue seems to be accessing the tmp directory from parallel builds at the same time (for caching in bootsnap), which locks things up.